src/Microsoft.DotNet.Build.Tasks/PackageFiles/notsupported.targets

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <PropertyGroup Condition="'$(GeneratePlatformNotSupportedAssembly)' == 'true'">
```

Comment:
FYI: behavior of an omitted value is what we'd expect: https://github.com/dotnet/buildtools/blob/3ed0bf65a3312946c959316a779ceb894cebdce7/src/common/CommandLine.cs#L357-L359

So that's more reason not to bother with using "Default"